client/flutter/lib/pages/question_index.dart

```diff
@@ -62,44 +66,79 @@ class _QuestionIndexPageState extends State<QuestionIndexPage> {
               )
             : SliverToBoxAdapter(
                 child: Padding(
-                  padding: const EdgeInsets.all(48.0),
-                  child: CupertinoActivityIndicator(),
-                )
-
-              )
+                padding: const EdgeInsets.all(48.0),
+                child: CupertinoActivityIndicator(),
+              ))
       ],
       title: widget.title,
     );
   }
 
-  Widget _buildQuestion(QuestionItem questionItem) {
-    return Padding(
-      padding: const EdgeInsets.all(8.0),
+  // flutter_html supports a subset of html: https://pub.dev/packages/flutter_html
```

Comment:
This comment seems lost in the file, should be at line 142?

Reply:
Yes, it should be at line 142.